app/src/main/java/com/cortado/mafkir/ui/time/TimePickerFragment.kt

```diff
@@ -0,0 +1,13 @@
+package com.cortado.mafkir.ui.time
+
+import android.app.Dialog
+import android.app.TimePickerDialog
+import android.os.Bundle
+import android.text.format.DateFormat
+import androidx.fragment.app.DialogFragment
+
+class TimePickerFragment(private val listener: TimePickerDialog.OnTimeSetListener) : DialogFragment() {
```

Comment:
Add option to pass the time to this dialog, so if you edit the time it will start with the time you previously had

Reply:
dn